lib/flex_commerce_api/json_api_client_extension/builder.rb

```diff
@@ -23,6 +24,17 @@ def find(args = {})
           klass.requestor.custom(:search, { request_method: :get }, params.merge(filter: @temp_search_criteria))
         end
       end
+
+      private
+
+      def apply_available_to_browse_filter
+        @temp_search_criteria[:filter]=search_filters.merge({ "available_to_browse" => { "eq" => true } })
+      end
+
+      def search_filters
+        search_filters = @temp_search_criteria[:filter] || {}
+        search_filters.is_a?(String) ? JSON.parse(search_filters) : search_filters
```

Comment:
because I noted that the params are being passed in as JSON already in the matalan front-end.
Not sure if it would be more sensible just to push back and tell them just to pass as a hash.
Just didnt want to introduce a breaking change for the time being.